dandiapi/api/mail.py

```diff
@@ -66,3 +74,33 @@ def send_ownership_change_emails(dandiset, removed_owners, added_owners):
     messages += [build_added_message(dandiset, added_owner) for added_owner in added_owners]
     with mail.get_connection() as connection:
         connection.send_messages(messages)
+
+
+# Email sent to the DANDI list when a new user logs in for the first time
+
+
+def registered_subject(user):
+    return f'DANDI: New user registered: {user.email}'
+
+
+def registered_message(user):
+    return f'A new user has logged in to the DANDI application: {user.email}.'
```

Comment:
let's fix up this message so that it is useful to the user. i'll come up with a draft.